product_template_tags/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
headers with the coding at least

Reply:
This is not necessary as this file will only contains import statement. If there would be a hook, I would understand but it's not the case.